gobblin-data-management/src/test/java/org/apache/gobblin/util/commit/SetPermissionCommitStepTest.java

```diff
@@ -72,11 +62,47 @@ public void tearDown() throws IOException {
   }
 
   @Test
-  public void testExecute() throws IOException {
-    this.step.execute();
+  public void testExecuteSingleLevel() throws IOException {
+    Path dir1 = new Path(ROOT_DIR, "dir1");
+    String owner = this.fs.getFileStatus(ROOT_DIR).getOwner();
+    String group = this.fs.getFileStatus(ROOT_DIR).getGroup();
+    FsPermission permission = new FsPermission(FsAction.ALL, FsAction.ALL, FsAction.ALL);
+    fs.mkdirs(dir1);
```

Comment:
let's pre-validate that `owner`, `group`, and `permission` aren't already at this point set w/ the expected values.  this disproves the possibility of `SetPermissionCommitStep` being a no-op.